docs/get-started/alerting.md

```diff
@@ -129,48 +227,65 @@ templates:
 ## Test alert expressions
 If you want to create custom templates, you can test the MetricsQL expressions for your custom template in the **Explore** section of PMM. Here you can also query any PMM internal database.
 
-
 To test expressions for custom templates:
 
 1. On the side menu in PMM, choose **Explore > Metrics**.
 2. Enter your expression in the **Metrics** field and click **Run query**.
 
 For example, to validate that a MongoDB instance is down, shut down a member of a three-node replica set, then check that the expression `{service_type="mongodb"}` returns **0** in **Explore > Metrics**.
 
-
 ### Add an alert rule
+
 After provisioning the resources required for creating Percona templated alerts, you are now ready to create your alert:
 
 1. Go to **Alerting > Alert Rules**, and click **New alert rule**.
 2. On the **Create alert rule** page, select the **Percona templated alert** option. If you want to learn about creating Grafana alerts instead, check our [Grafana's documentation](https://grafana.com/docs/grafana/latest/alerting/).
```

Comment:
You do NOT need to select the Percona templated option - it IS selected by default